src/views/HomeView.vue

```diff
@@ -10,10 +15,16 @@ const panelStore = usePanelsStore()
 
 <template>
   <main class="h-screen flex">
-    <div class="w-1/4 z-10 absolute">
-      <InformationPanel></InformationPanel>
+    <div class="z-10 absolute">
```

Comment:
it's good but IMO this should be out of the home page.
anyway let's keep it like this and refactor when we start the second page.